sample/src/main/res/layout/fragment_radio_buttons.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.core.widget.NestedScrollView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
What do you think of putting titles above the two sections to explain the two types of radio buttons on this screen ? Like `Simple radio buttons` and `Grouped radio buttons` or something else

Reply:
Agree, I'll put a `caption` title above both simple and grouped radio buttons. 
Since there was none with other components I followed the trend but I do feel the need for some captions too.